SourceX/controls/controller.cpp

```diff
@@ -22,9 +110,12 @@ ControllerButtonEvent ToControllerButtonEvent(const SDL_Event &event)
 	}
 
 #if HAS_KBCTRL == 1
-	result.button = KbCtrlToControllerButton(event);
-	if (result.button != ControllerButton_NONE)
-		return result;
+	KeyboardController *const keyboardController = KeyboardController::Get(event);
+	if (keyboardController != NULL) {

```

Comment:
Should we compare to `nullptr` for new code?

Reply:
sure but simple to replace, so don't want to enforce before we are consistent.